### PR TITLE
MOTECH-2330 Rename methods in EventListenerTree in event module.

### PR DESCRIPTION
**Pull request by jablonskidawid**

### CODE REVIEW

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by kmadej** [comment]
test this please

**Comment by kmadej** [comment]
@jablonskidawid please change the PR topic, 'rename methods ...'

File: platform/event/src/main/java/org/motechproject/event/listener/impl/EventListenerTree.java [hunk 1]
```diff
@@ -324,20 +324,20 @@ public void removeAllListeners(String beanName) {
...
             EventListenerTree child = listenerIterator.next();
-            if (child.containsListenersForBeanName(beanName) && child.allListenersEmpty()) {
+            if (child.removeListeners(beanName) && child.removeEmptyChildWithWildCardListeners()) {
                 listenerIterator.remove();
             }
         }
     }
 
-    private boolean allListenersEmpty() {
+    private boolean removeEmptyChildWithWildCardListeners() {
```
Comment (kmadej) [inline platform/event/src/main/java/org/motechproject/event/listener/impl/EventListenerTree.java]: Wildcard - its one word

**Comment by kmadej** [comment]
It's not easy to understand the purpose of methods in this class, however I think that so far this changes are enough for this ticket.

**Comment by kmadej** [comment]
test this please

**Pull request merged**